src/providers/departure-sort.service.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
+import { Storage } from '@ionic/storage';
+
+@Injectable()
+export class DepartureSortService {
+
+  constructor(public storage: Storage) {  }
+
+  validate(sort: string): string {
```

Comment:
The function I copied from the autorefresh service also returned a boolean stating whether the existing stored value was valid. But I couldn't see for what purpose it did that. So this function is more like a cleaner. It'll return the stored value if it's valid, but if not it'll just quietly substitute the default.

Reply:
This answers my previous question.